catalog/solr/catalog-solr-solrclient/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+    <OCD name="Catalog Solr Relevancy Ranking"
+         id="ddf.catalog.source.solr.rest.SolrRest">
+
+        <AD description="URL for retrieving Solr schema"
+            name="Solr Schema URL."
+            id="solrSchemaUrl"
+            type="String" default="${org.codice.ddf.system.protocol}${org.codice.ddf.system.hostname}:${org.codice.ddf.system.port}/solr/catalog/schema" />
+
+        <AD description="This parameter controls how quickly an increase in term frequency results in term-frequency saturation. The default value is 1.2. Lower values result in quicker saturation, and higher values in slower saturation."
+            name="k1" id="k1"
+            type="Float" default="1.2"/>
```

Comment:
❗️ The default in the blueprint is `1.25`.

Reply:
Fixed to match.